### PR TITLE
Posts for June and early July governance

### PR DESCRIPTION
**Pull request by valgrimm**
@XAMPPRocky Here is the combined post we discussed for your review. Happy to adjust.

### CODE REVIEW

File: posts/inside-rust/2020-06/2020-JuneJuly-governance-wg-meetings.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+[detailed minutes July 9](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.07.09.md)  <br>
+on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: <br>
+
+* There is a need for more clarity about why and how groups are formed, 
+    * what group categories mean, 
+    * how they relate to each other, 
+    * how they relate to the Rust Project
+    * and their level of decision-making power
+* Rust repository structure and permissions should ideally represent status of groups in the project
+* There will very likely be a proposal for Community Groups to have their own repositories outside of the Rust project and not be listed in
```
Comment (XAMPPRocky) [inline posts/inside-rust/2020-06/2020-JuneJuly-governance-wg-meetings.md:27]:
```suggestion
* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
```

File: posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: <br>
+
+* There is a need for more clarity about why and how groups are formed, 
+    * what group categories mean, 
+    * how they relate to each other, 
+    * how they relate to the Rust Project
+    * and their level of decision-making power
+* Rust repository structure and permissions should ideally represent status of groups in the project
+* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
+Rust materials. Instead, it is likely that, to avoid some community groups being perceived to be "more official",
```
Comment (nikomatsakis) [inline posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md:28]: Hmm, I'm not sure I'd say it's *likely* that the rust org would like to Mozilla's community portal. I don't know that I think linking to Mozilla's community portal makes sense, myself, given that the Rust community and Mozilla community can be somewhat distinct (if heavily overlapping). I guess I would say something like "One possibility might be to link to other community portals, such as Mozilla's community portal, where folks can list those groups."

File: posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    * what group categories mean, 
+    * how they relate to each other, 
+    * how they relate to the Rust Project
+    * and their level of decision-making power
+* Rust repository structure and permissions should ideally represent status of groups in the project
+* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
+Rust materials. Instead, it is likely that, to avoid some community groups being perceived to be "more official", 
+the Rust project would possibly only link to the [Mozilla Communities Portal](https://community.mozilla.org/), 
+and Rust community groups wishing to do outreach could can create profiles there, share events, and generally do promotion and recruitment
+* "Pre-FCP" likely going away
```
Comment (nikomatsakis) [inline posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md:31]:
```suggestion
* We discussed the idea of streamlining rfcbot procedures, as discussed in rust-lang/wg-governance#38.
```

How about this? I'm not sure people will have any idea what the existing sentence means.

File: posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md (hunk 2 above)
Comment (Manishearth) [reply posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md:28]: Yeah I find the mozilla mention here to be out of place. It's fine as an example, but we prob shouldn't be linking there

Comment (Manishearth) [reply posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md:28]: I guess we _could_ use that portal as a hosting space. But it seems unlikely.

Comment (valgrimm) [reply posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md]: Have adjusted post accordingly. Also revised based on Zulip-based input from XAMPPRocky about where to put minutes link.

**Comment by valgrimm** [comment]
Added comments from nmatsakis and XAMPPRocky to the post as well as fixed an error in the title. 
However, I can see that checks have failed. 

> thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }', src/posts.rs:48:21
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> ##[error]Process completed with exit code 101.

Any suggestions as to how I can fix this so the pull request can be merged by someone with access?

**Comment by XAMPPRocky** [comment]
@valgrimm Your file name needs to start with a full date in `YYYY-MM-DD` format to work properly. E.g `2020-01-01-title.md`

**Comment by valgrimm** [comment]
Thank you XAMPPRocky! All set now. Next post will also be batched.

**Comment by XAMPPRocky** [comment]
Going to close this since it's out of date now.

**Pull request closed without merging**